lib/membrane_webrtc_plugin/endpoint_bin.ex

```diff
@@ -228,7 +228,9 @@ defmodule Membrane.WebRTC.EndpointBin do
     %Track{ssrc: ssrc, encoding: encoding, rtp_mapping: mapping, extmaps: extmaps} =
       Map.fetch!(state.outbound_tracks, track_id)
 
-    rtp_extension_mapping = Map.new(extmaps, &Extension.as_rtp_mapping(state.extensions, &1))
+    rtp_extension_mapping =
+      extmaps
+      |> Map.new(&Extension.as_rtp_mapping(state.extensions, &1))
```

Comment:
We should avoid piping only once

```suggestion
    rtp_extension_mapping = Map.new(extmaps, &Extension.as_rtp_mapping(state.extensions, &1))
```